src/components/Cardlist.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useFetchCardsData } from '../hooks/CardContainer';
+import '../style/cardlist.css';
+import searchIcon from '../images/Search.svg';
+import noCardImg from '../images/noCardImg.png';
+
+const Cardlist = () => {
+  const data = useFetchCardsData();
+  const handleImageError = (e) => {
+    e.target.src = { noCardImg };
+  };
+
+  const handleSearch = (event) => {
+    event.preventDefault();
+  };
```

Comment:
```suggestion
  const handleImageError = (e) => {
    e.target.src = { noCardImg };
  };

  const handleSearch = (e) => {
    e.preventDefault();
  };
```

이 부분은 `e` 혹은 `event` 하나로 통일성있게 관리해주는 것이 좋습니다.

둘 중 어떤 변수명을 사용하느냐는 크게 중요한 부분이 아니니 통일성만 신경써주시면 감사할 것 같습니다!

Reply:
체크해주셔서 갑사합니다 !